web/src/app/components/Products/Products.tsx

```diff
@@ -1,16 +1,14 @@
 import * as React from 'react';
+import { Book, Product, Amount } from '../../metarpheus/model-ts';
```

Comment:
Suggerimento: aggiungi `export * from '../metarpheus/model-ts'` nel file `model/index.js`: così tutte le volte che vuoi importare dal data model puoi fare `from 'model'` sia che venga da metarpheus che sia stato definito da te in `models.js`.